.rubocop.yml

```diff
@@ -4,6 +4,7 @@ AllCops:
     - features/**/*
     - db/**/*
     - bundle_bin/**/*
+    - vendor/**/*
```

Comment:
[note] The RuboCop's default config also excludes `vendor/**/*`:

https://github.com/rubocop/rubocop/blob/84ac070f33e65349604a35562441f0b6510a12e0/config/default.yml#L66